lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -13,19 +13,23 @@
 import asyncio
 import logging
 
+from magma.common.grpc_client_manager import GRPCClientManager
 from magma.common.sentry import sentry_init
 from magma.common.service import MagmaService
-from magma.common.streamer import StreamerClient
 
+from .client import SubscriberDBCloudClient
 from .processor import Processor
 from .protocols.diameter.application import base, s6a
 from .protocols.diameter.server import S6aServer
 from .rpc_servicer import SubscriberDBRpcServicer
 from .subscription_profile import get_default_sub_profile
-from .streamer_callback import SubscriberDBStreamerCallback
 from .store.sqlite import SqliteStore
 from .protocols.s6a_proxy_servicer import S6aProxyRpcServicer
 from lte.protos.mconfig import mconfigs_pb2
+from lte.protos.subscriberdb_pb2_grpc import SubscriberDBCloudStub
+
+DEFAULT_SUBSCRIBERDB_SYNC_INTERVAL = 180
```

Comment:
I don't like having config defaults. If it's unavoidable then fine, but I think hard-coding defaults makes things brittle and causes unexpected values. Instead, we can just provide the default value through the yml file